src/_tutorials/dart-vm/cmdline.md

```diff
@@ -252,19 +257,21 @@ The standard I/O streams are defined at the top level of the dart:io library,
 | <a href="{{site.dart_api}}/dart-io/stdin.html" target="_blank">stdin</a> | The standard input |
 {: .table }
 
-Import the dart:io library as follows:
+Import the dart:io and dart:async libraries as follows:
 
 {% prettify dart %}
 import 'dart:io';
+import 'dart:async';
 {% endprettify %}
 
-Only command-line applications, not web applications,
-can use the dart:io library.
+Only command-line applications, not web applications, can use the dart:io library.
+The `async`, `await`, and `await for` keywords are built into the Dart language,
+but you must import dart:async to use Futures or Streams.
```

Comment:
use.... -> refer to the Future or Stream class.
?

(You can use them behind the scenes, as long as you don't use the types explicitly.)